climada/hazard/base.py

```diff
@@ -513,12 +513,14 @@ def from_xarray_raster(
 
             Default values are:
 
-            * ``date``: The ``event`` coordinate interpreted as date
+            * ``date``: The ``event`` coordinate interpreted as date or ordinal, or
+              zeros if that fails (which will issue a warning).
             * ``fraction``: ``None``, which results in a value of 1.0 everywhere, see
               :py:meth:`Hazard.__init__` for details.
             * ``hazard_type``: Empty string
             * ``frequency``: 1.0 for every event
-            * ``event_name``: String representation of the event time
+            * ``event_name``: String representation of the event date or empty strings
+              if that fails (which will issue a warning).
```

Comment:
Just to be sure I understand: if the data contains a field 'name', it will be ignored by default?

Reply:
This list only specifies the *default* values. These are chosen if the dataset does *not*  contain a field `event_name` OR the user chose to ignore it via `read_xarray_raster(..., data_vars=dict(event_name=""))`